cfme/tests/cloud/test_tag_mapping.py

```diff
@@ -41,8 +45,36 @@ def refresh_provider(provider):
     return True
 
 
+@pytest.fixture(params=['instances', 'images'])
+def tag_mapping_items(request, appliance, provider):
+    entity_type = request.param
+    collection = getattr(appliance.collections, 'cloud_{}'.format(entity_type))
+    collection.filters = {'provider': provider}
+    view = navigate_to(collection, 'AllForProvider')
+    name = view.entities.get_first_entity().name
+    try:
+        mgmt_item = (
+            provider.mgmt.get_template(name)
+            if entity_type == 'images'
+            else provider.mgmt.get_vm(name)
+        )
+    except ImageNotFoundError:
+        msg = 'Failed looking up template [{}] from CFME on provider: {}'.format(name, provider)
+        logger.exception(msg)
+        pytest.skip(msg)
+    return collection.instantiate(name=name, provider=provider), mgmt_item, entity_type
+
+
+@pytest.fixture
+def tag_components():
+    # Return tuple with random tag_label and tag_value
+    return ('tag_label_{}'.format(fauxfactory.gen_alphanumeric()),
+            'tag_value_{}'.format(fauxfactory.gen_alphanumeric()))
+
```

Comment:
@mshriver 
Why are we ```tag_components``` is  fixture? We are not even inheriting any other fixture or it is not doing any setup.
Can we define these strings globally?

Reply:
Good point @ganeshhubale.  Didn't want to refactor these commits too much myself, but this is a good change to make. Thanks!